docs/reference/metrics.cue

```diff
@@ -1,42 +1,1072 @@
+	type:        "gauge"
+	tags:        _apache_metrics_tags & {
+		state: {
+			description: "The state of the connection"
+			required:    true
+			examples: ["closing", "keepalive", "total", "writing"]
+		}
+	}
+}
+_apache_cpu_load: {
+	description:   "The current CPU of the Apache server."
+	relevant_when: "`ExtendedStatus On`"
+	type:          "gauge"
+	tags:          _apache_metrics_tags
+}
+_apache_cpu_seconds_total: {
+	description:   "The CPU time of various Apache processes."
+	relevant_when: "`ExtendedStatus On`"
+	type:          "counter"
+	tags:          _apache_metrics_tags & {
+		state: {
+			description: "The state of the connection"
+			required:    true
+			examples: ["children_system", "children_user", "system", "user"]
+		}
+	}
+}
+_apache_duration_seconds_total: {
+	description:   "The amount of time the Apache server has been running."
+	relevant_when: "`ExtendedStatus On`"
+	type:          "counter"
+	tags:          _apache_metrics_tags
+}
+_apache_scoreboard: {
+	description: "The amount of times various Apache server tasks have been run."
+	type:        "gauge"
+	tags:        _apache_metrics_tags & {
+		state: {
+			description: "The connect state"
+			required:    true
+			examples: ["closing", "dnslookup", "finishing", "idle_cleanup", "keepalive", "logging", "open", "reading", "sending", "starting", "waiting"]
+		}
+	}
+}
+_apache_sent_bytes_total: {
+	description:   "The amount of bytes sent by the Apache server."
+	relevant_when: "`ExtendedStatus On`"
+	type:          "counter"
+	tags:          _apache_metrics_tags
+}
+_apache_up: {
+	description: "If the Apache server is up or not."
+	type:        "gauge"
+	tags:        _apache_metrics_tags
+}
+_apache_uptime_seconds_total: {
+	description: "The amount of time the Apache server has been running."
+	type:        "counter"
+	tags:        _apache_metrics_tags
+}
+_apache_workers: {
+	description: "Apache worker statuses."
+	type:        "gauge"
+	tags:        _apache_metrics_tags & {
+		state: {
+			description: "The state of the worker"
+			required:    true
+			examples: ["busy", "idle"]
+		}
+	}
+}
+
+// Container metrics
+_vector_communication_errors_total: {
+	description: "The total number of errors stemming from communication with the Docker daemon."
+	type:        "counter"
+	tags:        _component_tags
+}
+
+_vector_container_events_processed_total: {
+	description: "The total number of container events processed."
+	type:        "counter"
+	tags:        _component_tags
+}
+
+_vector_container_metadata_fetch_errors_total: {
+	description: "The total number of errors caused by failure to fetch container metadata."
+	type:        "counter"
+	tags:        _component_tags
+}
+
+_vector_containers_unwatched_total: {
+	description: "The total number of times Vector stopped watching for container logs."
+	type:        "counter"
+	tags:        _component_tags
+}
+
+_vector_containers_watched_total: {
```

Comment:
I'd still prefer that these internal metrics be namespaced under `components: *: telemetry: metrics` for a few reasons:

1. They should not be used outside of the `components: *: telemetry: metrics` namespace.
2. It keeps the global namespace clean.

It aligns intent better with the usage.